cmd/storage.go

```diff
@@ -158,8 +158,13 @@ var storageListCmd = &cobra.Command{
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
 
+		if componentName == "" {
+			fmt.Println("No component selected")
+			os.Exit(1)
+		}
+
 		storageList, err := storage.List(client, componentName, applicationName)
-		checkError(err, "Failed to list storage")
```

Comment:
Why has been this message removed?

Reply:
@kadel It was hiding the underlying error without verbose flag enabled

Reply:
That was kind of the point of that.
But ok, lets leave it as it is for now.